sdk/src/Services/DynamoDBv2/Custom/DocumentModel/Table.cs

```diff
@@ -532,7 +545,7 @@ internal static Table CreateTableFromItemStorageConfig(IAmazonDynamoDB client, T
 
                 table.Attributes.Add(new AttributeDefinition { AttributeName = property.AttributeName, AttributeType = PrimitiveToScalar(primitiveType) });
             }
-            
```

Comment:
It would be great if the SDK had editor config support to highlight these // @normj